X16 Reference - 03 - BASIC.md

```diff
@@ -419,18 +419,18 @@ The numeric constants parser supports both hex (`$`) and binary (`%`) literals,
 
 The size of hex and binary values is only restricted by the range that can be represented by BASIC's internal floating point representation.
 
-#### LOAD into VRAM
+#### LOAD into HIRAM
```

Comment:
I think it makes sense to keep this section focused on loading directly to VRAM as this is indeed a new concept for X16 vs anyone with CBM experience....

Reply:
Enhanced LOAD functionality:
1) user-specified LOAD address: (LOAD supports syntax LOAD "myfile.prg",device,bank,address)
 - ignores the 2-byte PRG header address and loads the file into memory at the address specified in the final argument. If the address is in the HiRam bank window, LOAD will automatically extend the data into the next banks as necessary.
Headerless (binary) LOAD:
BLOAD and BVLOAD are available as "binary load" equivalents for loading files which do not have the 2-byte PRG header. These commands will not skip the first two bytes when loading. 
Right after VRAM loading.

Reply:
see also https://github.com/commanderx16/x16-rom/pull/312